RecoLocalCalo/EcalRecProducers/plugins/alpaka/AmplitudeComputationCommonKernels.h

```diff
@@ -76,8 +76,8 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::ecal::multifit {
           // for accessing input arrays
           int const inputTx = ch >= nchannelsEB ? idx.global - nchannelsEB * nsamples : idx.global;
           // eb is first and then ee
-          auto const* digis_in = ch >= nchannelsEB ? digisDevEE.data()->data() : digisDevEB.data()->data();
-          auto const gainId = ecalMGPA::gainId(digis_in[inputTx]);
+          auto const& digis_in = ch >= nchannelsEB ? digisDevEE.data() : digisDevEB.data();
```

Comment:
the span is constructed on the fly, so there is no advantage to using a reference.
can you remove the & for consistency with the other changes you made?